server/data/placementRequestClient.test.ts

```diff
@@ -33,8 +32,78 @@ describeClient('placementRequestClient', provider => {
     placementRequestClient = new PlacementRequestClient(token)
   })
 
+  describe('createBooking', () => {
+    it('creates and returns a booking', async () => {
+      const placementRequest = cas1PlacementRequestDetailFactory.build()
+      const bookingConfirmation = newPlacementRequestBookingConfirmationFactory.build()
+      const newPlacementRequestBooking = newPlacementRequestBookingFactory.build()
+
+      await provider.addInteraction({
+        state: 'Server is healthy',
+        uponReceiving: 'A request to create a booking from a placement request',
+        withRequest: {
+          method: 'POST',
+          path: paths.placementRequests.booking({ placementRequestId: placementRequest.id }),
+          body: newPlacementRequestBooking,
+          headers: {
+            authorization: `Bearer ${token}`,
+          },
+        },
+        willRespondWith: {
+          status: 200,
+          body: bookingConfirmation,
+        },
+      })
+
+      const result = await placementRequestClient.createBooking(placementRequest.id, newPlacementRequestBooking)
+
+      expect(result).toEqual(bookingConfirmation)
+    })
+  })
+
+  describe('bookingNotMade', () => {
+    it('makes a POST request to the booking not made endpoint', async () => {
+      const placementRequestId = faker.string.uuid()
+      const body = {
+        notes: 'some notes',
+      }
+      const bookingNotMade = bookingNotMadeFactory.build()
+
+      await provider.addInteraction({
+        state: 'Server is healthy',
+        uponReceiving: 'A request to mark a placement request as not booked',
+        withRequest: {
+          method: 'POST',
+          path: paths.placementRequests.bookingNotMade({ placementRequestId }),
+          body,
+          headers: {
+            authorization: `Bearer ${token}`,
+          },
+        },
+        willRespondWith: {
+          status: 200,
+          body: bookingNotMade,
+        },
+      })
+
+      const result = await placementRequestClient.bookingNotMade(placementRequestId, body)
+
+      expect(result).toEqual(bookingNotMade)
+    })
+  })
+})
+
+describeCas1NamespaceClient('Cas1PlacementRequestClient', provider => {
```

Comment:
We don't need to split the namespaced client tests from the non-namespaced ones anymore I believe. Might be worth cleaning this up as the file is being touched.

Reply:
True -- I'll see if I can clean that up in a separate PR.